### PR TITLE
add text direction classification model (mobilenetv3_small_100)

### PR DESCRIPTION
**Pull request by HaoyangLee**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [x] You have added unit tests

## Motivation

- add mobilenetv3_small_100 as text direction classification model.
- achieve evaluation accuracy 94.59%, higher than paddleocr 94.56%.
- docs will be updated later.

### CODE REVIEW

File: configs/cls/mv3/cls_mv3.yaml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    transform_pipeline:
+      - DecodeImage:
+          img_mode: BGR
+          to_float32: False
+      - ClsLabelEncode:
+          label_list: *label_list
+      - RecResizeImg: # different from paddle (paddle converts image from HWC to CHW and rescale to [-1, 1] after resize.
+          image_shape: [48, 192] # H, W
+          padding: False # aspect ratio will be preserved if true.
+      - NormalizeImage:  # different from paddle (paddle wrongly normalize BGR image with RGB mean/std from ImageNet for det, and simple rescale to [-1, 1] in rec.
```
Comment (liangxhao) [inline configs/cls/mv3/cls_mv3.yaml:85]: rec -> cls

Comment (HaoyangLee) [reply configs/cls/mv3/cls_mv3.yaml]: removed these redundant comment messages

**Review by liangxhao — APPROVED** [review]

File: mindocr/models/heads/cls_mv3_head.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+class ClsHead(nn.Cell):
+    """
+    Text direction classification head.
+    """
+    def __init__(self, in_channels: int, out_channels: int, num_classes: int):
+        super().__init__()
+        self.pool = GlobalAvgPooling()
+        self.classifier = nn.SequentialCell([
+            nn.Dense(in_channels, out_channels),
```
Comment (zhtmike) [inline mindocr/models/heads/cls_mv3_head.py:15]: `out_channels` usually corresponds to the real output channels (or the `num_classes` in this case ), it is better to name it as "hidden_channels"

**Review by zhtmike — APPROVED** [review]

File: mindocr/models/heads/cls_mv3_head.py (hunk 2 above)
Comment (HaoyangLee) [reply mindocr/models/heads/cls_mv3_head.py]: updated

**Comment by HaoyangLee** [comment]
Issue #230

**Pull request merged**